rero_ils/modules/cli/utils.py

```diff
@@ -1691,3 +1692,46 @@ def add_cover_urls(verbose):
 def delete_loans_created(verbose, hours):
     """Delete loans with state CREATED."""
     return task_delete_loans_created(verbose=verbose, hours=hours)
+
+
+@utils.command()
+@click.argument('outfile', type=click.File('w'))
```

Comment:
`file` ?

Reply:
I allways make a difference between `infile`, `outile` etc. This name is seen by `--help`.